plugins/inputs/jolokia/README.md

```diff
@@ -17,7 +17,21 @@
   # [inputs.jolokia.proxy]
   #   host = "127.0.0.1"
   #   port = "8080"
+  
+  ## Optional http timeouts
+  ##
+  ## ResponseHeaderTimeout, if non-zero, specifies the amount of
+  ## time to wait for a server's response headers after fully
+  ## writing the request (including its body, if any). This
+  ## time does not include the time to read the response body.
+  ## See http.Transport.ResponseHeaderTimeout
+  # responseHeaderTimeout = 3
```

Comment:
these should match the time format ("3s" and "4s"), and they should be response_header_timeout and client_timeout.